sync_issues_to_jira/sync_issue.py

```diff
@@ -458,4 +457,21 @@ def _get_jira_comment_body(gh_comment, body=None):
 
 def _get_jira_label(gh_label):
     """ Reformat a github API label item as something suitable for JIRA """
+    # ignore status, resolution and platform: windows labels
+    if _check_issue_label(gh_label["name"]) is None:
+        return
     return gh_label["name"].replace(" ", "-")
+
+def _check_issue_label(label):
+    """
+    Ignore labels that start with "Status:" and "Resolution:". These labels are
+    mirrored from Jira issue and should not be mirrored back as labels
+    """
+    ignore_prefix = ("status:", "resolution:")
+    if label.lower().startswith(ignore_prefix):
+        return None
+    
+    if label.lower() == "platform: windows":
```

Comment:
The DRY principle (don't repeat yourself) tells us that any piece of information in a system should have a single source of truth.

In this case this "special" label is encoded in two places, here and on line 74.

What about putting a constant near the top of the file, like this:

```python
# Some GitHub labels are mapped directly to JIRA components
GH_LABEL_TO_JIRA_COMPONENT = {
    "platform: windows": "windows platform"
}
```

and then can refer to this dict elsewhere, like `if label.lower() in GH_LABEL_TO_JIRA_COMPONENT:`